src/Sarif.FunctionalTests/SarifConverterTestData/AndroidStudio/singleIssue.xml.sarif

```diff
@@ -27,17 +39,7 @@
                   "startLine": 242
                 }
               },
-              "logicalLocation": [
-                {
-                  "name": "northwindtraders-droid-2",
-                  "kind": "android-module"
-                },
-                {
-                  "name": "com.northwindtraders.droid.model",
-                  "kind": "package"
-                }
-              ],
-              "fullyQualifiedLogicalName": "northwindtraders-droid-2\\com.northwindtraders.droid.model"
+              "logicalLocation": "northwindtraders-droid-2\\com.northwindtraders.droid.model"
```

Comment:
I'm not sure any of these even are actually logical locations, now that I think of it. all this looks like nesting information. a logical location refers to a logical organizational scheme within a discrete component. now sure that's what's happening here.